zerver/tests/test_soft_deactivation.py

```diff
@@ -370,6 +370,7 @@ def send_fake_message(message_content: str, stream: Stream) -> Message:
             recipient = stream.recipient
             message = Message(
                 sender=sender,
+                realm=realm,
                 recipient=recipient,
                 content=message_content,
                 date_sent=timezone_now(),
```

Comment:
This helper aso seems like it has the potential to be rewritten as just using the standard `self.send_stream_message` code path.

Reply:
Hmm, here i don't think so because the manual `Message` creation seems to have a purpose, as opposed to using `send_stream_message`:
```
        # Test that add_missing_messages() in simplest case of adding a
        # message for which UserMessage row doesn't exist for this user.
        sent_message = send_fake_message("Test message 1", stream)
```

Reply:
OK; we should perhaps add a comment to document that.

Reply:
Sure, pushed a prep commit